crates/tx5-connection/src/conn.rs

```diff
@@ -394,7 +394,7 @@ async fn con_task_attempt_webrtc(
 ) -> AttemptWebrtcResult {
     use AttemptWebrtcResult::*;
 
-    let timeout_dur = task_core.config.signal_config.max_idle;
```

Comment:
This `max_idle` is set to the same value as `Config::timeout` by the calling code.